plugins/gzip/gzip.cc

```diff
@@ -480,8 +480,8 @@ gzip_transformable(TSHttpTxn txnp, bool server, HostConfiguration *host_configur
 
   resp_status = TSHttpHdrStatusGet(bufp, hdr_loc);
 
-  // conservatively pick some statusses to compress
-  if (!(resp_status == 200 || resp_status == 404 || resp_status == 500)) {
+  // conservatively pick some statusses to compress. ToDo: Make this configurable?
+  if (resp_status != 200) {
```

Comment:
This seems like a good change to me, it's the probably safer default -- but is it possible that we see `200/OK` two times  here when the HttpSM is restarted?